sonic-utilities-tests/system_health_test.py

```diff
@@ -0,0 +1,113 @@
+        else:
+            state = MockerManager.STATE_RUNNING
+            stats = {}
+        MockerManager.counter += 1
+
+        return state, stats
+
+class MockerChassis(object):
+    counter = 0
+
+    def initizalize_system_led(self):
+        return
+
+    def get_status_led(self):
+        if MockerChassis.counter == 1:
+            MockerChassis.counter += 1
+            return "green"
+        else:
+            MockerChassis.counter += 1
+            return "red"
+
+import show.main as show
+
+class TestHealth(object):
+    @classmethod
+    def setup_class(cls):
+        print("SETUP")
+        os.environ["PATH"] += os.pathsep + scripts_path
+        os.environ["UTILITIES_UNIT_TESTING"] = "1"
+
+    def test_health_summary(self):
+        runner = CliRunner()
+        result = runner.invoke(show.cli.commands["system-health"].commands["summary"])
+        click.echo(result.output)
+        expected = """System is currently booting...\n"""
+        assert result.output == expected
+        result = runner.invoke(show.cli.commands["system-health"].commands["summary"])
+        expected = """System status summary\n\n  System status LED  red\n  Services:\n    Status: Not OK\n    Not Running: 'telemetry', 'snmp_subagent'\n  Hardware:\n    Status: OK\n"""
```

Comment:
please define the output properly.  multi line output should be multi line.

Reply:
@lguohan fixed it, please check it.